bridge/discord/discord.go

```diff
@@ -189,6 +191,18 @@ func (b *Bdiscord) Send(msg config.Message) (string, error) {
 		if len(msg.Username) > 32 {
 			msg.Username = msg.Username[0:32]
 		}
+		// if we have a global webhook for this Discord account, use it, but
+		// first set its channel to the message channel
+		// TODO: this isn't necessary if the last message from this webhook was
```

Comment:
Any thoughts on how you'd implement this check? I feel like it can't be as simple as checking `existingWebhook.ChannelID` via `existingWebhook, err := b.c.Webhook(wID)`, what with multiple gateways possibly fighting over the one webhook...

I'm open to this being out-of-scope for this PR, but curious if a need for locks might change this code :)

(I am still a beginner with golang)

Reply:
I'm open to suggestions here too.

> checking `existingWebhook.ChannelID` via `existingWebhook, err := b.c.Webhook(wID)`

This + appropriate locking (if needed?) would work.  Possibly even better (but slightly more risky) would be a mechanism to remember the last channel that the webhook used, which would avoid the extra API call to check the webhook.

Since this isn't trivial (to me at least), and it might end up causing _more_ total API calls rather than _less_, I'm inclined to leave it for a separate PR.